utils/wasm-builder/src/wasm_project.rs

```diff
@@ -211,7 +211,7 @@ impl WasmProject {
         // Copy original `Cargo.lock` if any
         let from_lock = self.original_dir.join("Cargo.lock");
         let to_lock = self.out_dir.join("Cargo.lock");
-        let _ = fs::copy(from_lock, to_lock);
+        drop(fs::copy(from_lock, to_lock));
```

Comment:
As for me, the previous one was better. Does it cause a warning?

Reply:
yes, it uses `-Wlet-underscore` in `rustflags`

Reply:
And what if we write:

```rust
_ = fs::copy(from_lock, to_lock);
```

Reply:
it still causes warning
```
warning: non-binding let on a type that implements `Drop`
   --> /home/.../gear/utils/wasm-builder/src/wasm_project.rs:214:9
    |
214 |         _ = fs::copy(from_lock, to_lock);
```